### PR TITLE
ACL 2022 ingestion

### PR DESCRIPTION
**Pull request by xinru1414**
test ingested acl-short and lnls

### CODE REVIEW

**Comment by xinru1414** [comment]
Test ingestion pdf and attachments aren't synced on the server.

Potential problems:
- Some `conference_details.yaml` file may not contain `editors` field
- `conference_details.yaml` files seen-so-far do not contain `doi` field and `sig` field
- `papers.yaml` files seen-so-far do not contain `pages`, `language` and `doi` field

**Comment by mjpost** [comment]
sig is optional

lack of editors is an error

doi is added by us later 

pages is a problem. we should let danilo know but for these volumes maybe we can add it programmatically

File: data/xml/2022.acl.xml
Comment (mjpost) [inline data/xml/2022.acl.xml:455]: It looks like you're not applying case protection at all? You have to call `normalize()` on the title (see old script for an example).

File: bin/ingest_aclpub2.py
Comment (mjpost) [inline bin/ingest_aclpub2.py:63]: Best to import these from `ingest.py`, just to reduce redundancy. (You probably have to move them first, since they're inside `main()`)

File: bin/ingest_aclpub2.py
Comment (mjpost) [inline bin/ingest_aclpub2.py:484]: This should be uncommented.

File: bin/ingest_aclpub2.py
Comment (mjpost) [inline bin/ingest_aclpub2.py:506]: This, too, could be used if present?

File: bin/ingest_aclpub2.py
Comment (xinru1414) [reply bin/ingest_aclpub2.py:493]: fixed

File: bin/ingest_aclpub2.py
Comment (xinru1414) [reply bin/ingest_aclpub2.py:484]: fixed

**Comment by crux82** [comment]
Dear all, 

this is just a message to remember that the acl-short-volume was only a trial. 
I will send the real one as soon as possible. 

Bests

Danilo

File: data/yaml/venues.yaml
Comment (mjpost) [inline data/yaml/venues.yaml:245]: I spoke with them and decided we can use this acronym. @xinru1414, can you fix the `\\`, and add the workshop URL: https://lcs2.iiitd.edu.in/CONSTRAINT-2022/

**Comment by crux82** [comment]
@xinru1414 Can you please take a look at 

https://preview.aclanthology.org/new_ingestion/volumes/2022.deelio-1/

The title has \\ and the name of Ivan Vulic should be escaped

**Comment by crux82** [comment]
There are still a couple of mentions of "Vuli'\{c}" at https://preview.aclanthology.org/new_ingestion/events/acl-2022/

can you fix them?

It is the last mention to the symbol \

**Comment by crux82** [comment]
Is it normal that the findings are included here:

https://preview.aclanthology.org/new_ingestion/events/acl-2022/

(half of the list)

**Comment by xinru1414** [comment]
> https://preview.aclanthology.org/new_ingestion/events/acl-2022/

@crux82 yes, see https://aclanthology.org/events/emnlp-2021/

**Comment by crux82** [comment]
Dear all, sorry but at 

https://preview.aclanthology.org/new_ingestion/events/acl-2022/

I do not see the following workshops:

- bigscience: Proceedings of BigScience Episode 5 - Workshop on Challenges & Perspectives in Creating Large Language Models
- csrr: Proceedings of the First Workshop on Commonsense Representation and Reasoning
- cmcl: Proceedings of the Workshop on Cognitive Modeling and Computational Linguistics
- Lchange: Proceedings of the 3rd Workshop on Computational Approaches to Historical Language Change

Am I missing something?

**Comment by mjpost** [comment]
CMCL and LChange have been added. That is just the joint event listing. You can always find the ingested volumes at, e.g., https://aclanthology.org/venues/lchange.

I don't see any "bioscience" or "csrr" venue at all. What are their Anthology venue IDs?

@xinru1414 can you adjust the "ingestion-date" tag on all imported volumes to 2022-05-15? That's a flag to `ingest.py`, and the date is meant to reflect the date the PDFs were made available.

**Comment by xinru1414** [comment]
> CMCL and LChange have been added. That is just the joint event listing. You can always find the ingested volumes at, e.g., https://aclanthology.org/venues/lchange.
> 
> I don't see any "bioscience" or "csrr" venue at all. What are their Anthology venue IDs?
> 
> @xinru1414 can you adjust the "ingestion-date" tag on all imported volumes to 2022-05-15? That's a flag to `ingest.py`, and the date is meant to reflect the date the PDFs were made available.

will do!

**Comment by xinru1414** [comment]
@crux82 @mjpost I ingested csrrr & bigscience.

**Comment by crux82** [comment]
Again in https://preview.aclanthology.org/new_ingestion/events/acl-2022/#2022-bigscience-1

the authors added "\" in the title and  "Matthias Gall\'e " in the editor

**Comment by crux82** [comment]
I noticed that in https://preview.aclanthology.org/new_ingestion/volumes/2022.acl-tutorials/

There are only links to the PDFs of the tutorial. 

However, the tutorial chairs added in the papers.yml the links about the material of the tutorial. Can you add them?
I think that these 8 links are extremely valuable for the participants...

**Comment by mjpost** [comment]
We don't have a way to link to arbitrary URLs from papers. The tutorials should be linked from the conference website.

**Comment by crux82** [comment]
Ah, we need to update the guide as we wrote that also URL can be used  :-(

I will update the tutorial chairs...

**Comment by crux82** [comment]
@mjpost In any case, how can someone add a link to  https://paperswithcode.com/paper/?acl=2021.acl-long.115

**Comment by mjpost** [comment]
These happen automatically when PwC crawls them.

**Comment by crux82** [comment]
Ok Thanks!

By the way, once the name/editors of bigscience are fixed, for me you can go online!

**Review by mjpost — APPROVED** [review]
LGTM. I'll merge once the last fix is approved.

**Comment by crux82** [comment]
Thank for the merge.

But all the pdfs of the frontmatters 0.pdf are missing

**Pull request merged**
